pkg/backends/otlp/internal/data/metric_request.go

```diff
@@ -1,6 +1,8 @@
 package data
 
 import (
+	"bytes"
+	"compress/gzip"
```

Comment:
I think this will be fine for now, but we will need to look at adopting lz4 compression to help improve performance.

Reply:
idk if there's choice for now as the OTLP doc is quite confusing,  [specification here](https://opentelemetry.io/docs/specs/otlp/#protocol-details) says it only need to support `gzip` to meet the minimal requirement for a OTLP server